Overhaul of README - All Repos and diagrams updated. (#194)
AdoptOpenJDK/TSC: README.md

1. **A Repository of professionally built, tested and supported OpenJDK binaries** - A place for end users to download professionally built and tested OpenJDK binaries, see [Support](https://adoptopenjdk.net/support.html) for our full support details.
1. **A Community of builders** - A place where those who build and test OpenJDK come together to share common code and practices.

> 1. **Infrastructure as Code** - To host, build, test and deploy variants of [OpenJDK](http://openjdk.java.net) (aka Java)! This infrastructure as code is designed to be usable by any person or organisation wishing to build a derivative build farm or parts of one.

Reviewer: ```suggestion
1. **Infrastructure as Code** - To host, build, test and deploy variants of [OpenJDK](http://openjdk.java.net) (aka Java)! This infrastructure as code is based around ansible and is designed to be usable by any person or organisation wishing to build a derivative build farm or parts of one.
```

![Source Repositories](images/Adopt_Build_Farm_Source_Repos.png)

When an OpenJDK variant is mercurial based or AdoptOpenJDK needs to maintain its own patches then we have a clone of that source:

* [openjdk-jdk8u](https://github.com/AdoptOpenJDK/openjdk-jdk8u)
* [openjdk-aarch64-jdk8u](https://github.com/AdoptOpenJDK/openjdk-aarch64-jdk8u)
* [openjdk-aarch32-jdk8u](https://github.com/AdoptOpenJDK/openjdk-aarch32-jdk8u)
* [openjdk-jdk9u](https://github.com/AdoptOpenJDK/openjdk-jdk9u)
* [openjdk-jdk10u](https://github.com/AdoptOpenJDK/openjdk-jdk10u)
* [openjdk-jdk11u](https://github.com/AdoptOpenJDK/openjdk-jdk11u)
* [openjdk-jdk12u](https://github.com/AdoptOpenJDK/openjdk-jdk12u)
* [openjdk-jdk13u](https://github.com/AdoptOpenJDK/openjdk-jdk13u)
* [openjdk-jdk14u](https://github.com/AdoptOpenJDK/openjdk-jdk14u)
* [openjdk-jdk15u](https://github.com/AdoptOpenJDK/openjdk-jdk15u)
* [openjdk-jdk16](https://github.com/AdoptOpenJDK/openjdk-jdk16)
* [openjdk-jdk](https://github.com/AdoptOpenJDK/openjdk-jdk)
* [openjdk-jfx](https://github.com/AdoptOpenJDK/openjdk-jfx)
* [openjdk-amber](https://github.com/AdoptOpenJDK/openjdk-amber)

> * [openjdk-amber-nightly](https://github.com/AdoptOpenJDK/openjdk-amber-nightly)

Reviewer: Not sure this one is kept up to date - The [JavaFutures](https://ci.adoptopenjdk.net/view/Build%20and%20Test%20Pipeline%20Calendar/job/JavaFutures/) job pulls direct from Skara repos now

* [openjdk10-binaries](https://github.com/AdoptOpenJDK/openjdk10-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 10 (HotSpot VM and Eclipse OpenJ9 VM).
* [openjdk11-binaries](https://github.com/AdoptOpenJDK/openjdk11-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 11 (HotSpot VM and Eclipse OpenJ9 VM).
* [openjdk12-binaries](https://github.com/AdoptOpenJDK/openjdk12-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 12 (HotSpot VM and Eclipse OpenJ9 VM).
* [openjdk13-binaries](https://github.com/AdoptOpenJDK/openjdk13-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 13 (HotSpot VM and Eclipse OpenJ9 VM).
* [openjdk14-binaries](https://github.com/AdoptOpenJDK/openjdk14-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 14 (HotSpot VM and Eclipse OpenJ9 VM).
* [openjdk15-binaries](https://github.com/AdoptOpenJDK/openjdk15-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 15 (HotSpot VM and Eclipse OpenJ9 VM).

> * [openjdk16-binaries](https://github.com/AdoptOpenJDK/openjdk16-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK 16 (HotSpot VM and Eclipse OpenJ9 VM).

Reviewer: Include [openjdk17-binaries](https://github.com/AdoptOpenJDK/openjdk17-binaries/releases) too? I'd be slightly tempted to mark which of these are now obsolete and no longer updated, but is it really worth it

* [openjdk-binaries](https://github.com/AdoptOpenJDK/openjdk-binaries/) - AdoptOpenJDK nightly and release binaries for AdoptOpenJDK.

### Upstream OpenJDK binary repositories

> These GitHub Repositories are where builds from the OpenjDK update projects (led by Red Hat) are published to. The API (and subsequently) website poll these repositories to make binaries available.

Reviewer: ```suggestion
These GitHub Repositories are where builds from the OpenJDK update projects (led by Red Hat) are published to. The API and (using the API) the website poll these repositories to make binaries available.
```

1. **OpenJDK SAP** - SAP source is built from e.g. [openjdk-sap-jdk10](https://github.com/SAP/SapMachine)

> 1. **OpenJDK OpenJFX** - GitHub mirror of OpenJDK JFX lives at AdoptOpenJDK - [openjdk-jfx](https://github.com/AdoptOpenJDK/openjdk-jfx)

Reviewer: This looks empty - should we remove the reference?

Our Jenkins [Pipelines](https://ci.adoptopenjdk.net/view/Pipelines/) ([source code](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines)) build
binaries and then push them through a test pipeline (**NOTE** Test pipeline is skipped for certain unstable combinations).

**NOTE:** Once the Test jobs have stabilised, the pipeline described below will change so that only tested binaries are released to Nightly and Release repositories for consumption by the Website and/Or API.  See [The Quality Bar Discussion](https://github.com/AdoptOpenJDK/openjdk-tests/issues/186)
for details.

#### 3.1 Build

> [Builds](https://ci.adoptopenjdk.net/) are run on the [Supported Platforms](https://docs.google.com/spreadsheets/d/1T_sYpMe1y2cSdiP83HA2VWnTH6B-Nbl3ppss0gEpACc/edit?usp=sharing). The Jenkins leader

Reviewer: That document only has up to JDK10 so sugget using the link to that information on our website
```suggestion
[Builds](https://ci.adoptopenjdk.net/) are run on the [Supported Platforms](https://adoptopenjdk.net/supported_platforms.html). The Jenkins leader
```

sends the build jobs to the Jenkins followers based on a tagging system configured in the Jenkins jobs, e.g. `centos6&&x64&&build`. See [openjdk-build](https://github.com/AdoptOpenJDK/openjdk-build) and [openjdk-infrastructure](https://github.com/AdoptOpenJDK/openjdk-infrastructure) for details.

#### 3.2 Test

Note that other OpenJDK binaries (such as the [openjdk8-upstream-binaries](https://github.com/AdoptOpenJDK/openjdk8-upstream-binaries/) and [openjdk11-upstream-binaries](https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/)) can be put through pipelines entering at this Test phase.

> [Tests](https://ci.adoptopenjdk.net/view/OpenJDK%20tests/) are run on the

Reviewer: Link does not exist - jobs are split across various views in jenkins depending on the suite
```suggestion
Tests are run on the
```

> [Supported Test Platforms](https://docs.google.com/spreadsheets/d/1X4CCfvMoCgEavRbvejHrTvPnqj37MB-_C6LB6b8Akkc/edit?usp=sharing). The Jenkins leader

Reviewer: Hmm this is a different supported platforms doc to the earlier one, but it also doesn't mention anything later than JDK10 so not sure it's useful to link to as it's clearly not being kept up to date with relevant info.

Author: New issue created for this

**NOTE** Future versions of this workflow will show the status of testing and meta data about how the binary was built.

> 1. **Binaries are deployed** - Using the [OpenJDK Release Tool](https://ci.adoptopenjdk.net/view/Tooling/job/openjdk_release_tool/) (from the

Reviewer: ```suggestion
1. **Binaries are deployed** - Using the [OpenJDK Release Tool](https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/refactor_openjdk_release_tool/) (from the
```

[openjdk-webiste-backend](https://github.com/AdoptOpenJDK/openjdk-website-backend) project) in order to:
    1. deploy them to the various binary repos:
[openjdk-website-backend](https://github.com/AdoptOpenJDK/openjdk-website-backend) project) in order to:
    1. deploy them to the various binary repositories:
    ([openjdk8-binaries](https://github.com/AdoptOpenJDK/openjdk8-binaries/),
    [openjdk9-binaries](https://github.com/AdoptOpenJDK/openjdk9-binaries/),
    [openjdk10-binaries](https://github.com/AdoptOpenJDK/openjdk10-binaries/),
    [openjdk11-binaries](https://github.com/AdoptOpenJDK/openjdk11-binaries/),
    [openjdk12-binaries](https://github.com/AdoptOpenJDK/openjdk12-binaries/),
    [openjdk13-binaries](https://github.com/AdoptOpenJDK/openjdk13-binaries/),
    [openjdk14-binaries](https://github.com/AdoptOpenJDK/openjdk14-binaries/),
    [openjdk15-binaries](https://github.com/AdoptOpenJDK/openjdk15-binaries/),
    [openjdk-binaries](https://github.com/AdoptOpenJDK/openjdk-binaries/)).

> 1. **Binaries made available** - Binaries are made available for download via the [website](https://www.adoptopenjdk.net)

Reviewer: Should we add a section here mentioning installer/signing since we've got most of the rest?

Author: Added a TODO section for this

> 1. **Binaries made available** - Binaries are made available for download via the [website](https://www.adoptopenjdk.net) and [api](https://api.adoptopenjdk.net) gateway. See [openjdk-website](https://github.com/AdoptOpenJDK/openjdk-website), [openjdk-api](https://github.com/AdoptOpenJDK/openjdk-api) and [openjdk-api-java-client](https://github.com/AdoptOpenJDK/openjdk-api-java-client) projects for more details.

Reviewer: Docker too?

Author: fixed.
